android/assets/jsons/translationsByLanguage/Romanian.properties

```diff
@@ -1462,11 +1314,9 @@ Victory Types =
 Workers = 
 
 # Other civilopedia things
- # Requires translation!
-Nations = 
+Nations = Națiuni
 Promotions = Promoţii
- # Requires translation!
-Available for [unitTypes] = 
+Available for [unitTypes] = Valabil pentru [unitType]
```

Comment:
**TYPO:** this line should be
`Available for [unitTypes] = Valabil pentru [unitTypes]`

Reply:
Thanks